docs/Technical Reference.md

```diff
@@ -2,110 +2,102 @@ This page provides a technical reference on certain advanced configuration optio
 
-1. *Sufficient permissions* - Creating a CloudFormation stack requires sufficient permissions, such as the
-`AdministratorAccess` policy
-1. *A properly configured S3 bucket* where you will search, store, and browse data with Quilt. It is **highly recommended** that you back up said S3 bucket before proceeding.
-   * [Enable object versioning](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/enable-versioning.html) on the bucket
-   * Set a [CORS policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html#how-do-i-enable-cors), similar to the following, on the bucket:
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+    <CORSRule>
+        <AllowedOrigin>https://yourcompanydomain.com</AllowedOrigin>
+        <AllowedMethod>GET</AllowedMethod>
+        <AllowedMethod>HEAD</AllowedMethod>
+        <AllowedMethod>PUT</AllowedMethod>
+        <AllowedMethod>POST</AllowedMethod>
+        <AllowedHeader>*</AllowedHeader>
+        <MaxAgeSeconds>3000</MaxAgeSeconds>
+    </CORSRule>
+    </CORSConfiguration>
+    ```
 
-      ```xml
-      <?xml version="1.0" encoding="UTF-8"?>
-      <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
-      <CORSRule>
-          <AllowedOrigin>https://yourcompanydomain.com</AllowedOrigin>
-          <AllowedMethod>GET</AllowedMethod>
-          <AllowedMethod>HEAD</AllowedMethod>
-          <AllowedMethod>PUT</AllowedMethod>
-          <AllowedMethod>POST</AllowedMethod>
-          <AllowedHeader>*</AllowedHeader>
-          <MaxAgeSeconds>3000</MaxAgeSeconds>
-      </CORSRule>
-      </CORSConfiguration>
-      ```
+    > Note: bucket CORS does not grant permissions of any kind.
+    > `AllowedMethod` actions are only available to IAM users or roles with sufficient permissions.
 
-      The above policy makes it possible for your users to browse buckets on the web.
-      > Note: bucket CORS does not grant permissions of any kind.
-      > `AllowedMethod`s are only callable by IAM users or roles with sufficient permissions.
-1. *ARN for server certificate* - We require HTTPS for access to the Quilt web catalog. You may use the CloudFront domain output by CloudFormation. Alternatively, if you wish to use a custom domain, create or upload an [SSL/TLS server certificate](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_server-certs.html).
-1. *ARNs for search users* - Only select users that you specifcy can search S3. Search users can view contents for any bucket object, so only add users that you trust to this list.
+    Note the `AllowedOrigin` field. This should be parameterized with the domain you will host your catalog from.
 
-### Permissions
+    If you are going to use Quilt T4 with a new bucket, create the bucket now, and set these policies as part of the flow for bucket creation.
```

Comment:
see above; what this section really needs to state is that the template will modify the bucket properties: CORS, enable object versioning

Reply:
@akarve To our knowledge that doesn't actually happen?